layer.py

```diff
@@ -171,16 +180,16 @@ def cloud_action(self):
     def cloud_action(self, action):
         self._cloud_action = action
 
-    def photo_naming(self, field_name: str) -> str:
-        return self._photo_naming.get(
+    def attachment_naming(self, field_name, attachment_type: str) -> str:
```

Comment:
We either get the `attachment_type` as an int already.

Reply:
See my suggestion in `https://github.com/opengisch/libqfieldsync/pull/41#discussion_r1089725178`, then it would be of type `AtachmentType`.